examples/utils/bundle-size-tests/webpack.config.js

```diff
@@ -11,6 +11,9 @@ const DuplicatePackageCheckerPlugin = require('duplicate-package-checker-webpack
 module.exports = {
   entry: {
     'container': './src/container',
+    'map': './src/map',
+    'matrix': './src/matrix',
+    'odspDriver': './src/odspDriver',
```

Comment:
I was going to add this then npm i completely broke 🙃 thanks for adding it

Reply:
Been there with those npm install issues! Not sure what other packages make sense to add, but I think this captures a good approximation of what we consume in office scenarios